src/sentry/constants.py

```diff
@@ -232,6 +234,41 @@ def get_all_languages():
 
 NATIVE_UNKNOWN_STRING = '<unknown>'
 
+# to go from a marketing page slug like /for/android/ to the integration id
+# (in _platforms.json), for looking up documentation urls, etc.
+MARKETING_SLUG_TO_INTEGRATION_ID = {
+    # java
+    "java": "java",
+    "kotlin": "java",
+    "scala": "java",
+    "android": "java-android",
+
+    # TODO: add more...
+}
```

Comment:
So I feel like this one is useful for going from `/for/flask` to `python-flask` (the integration name in `platform.json`).